docs/manual/getting_started/installation/platforms.rst

```diff
@@ -0,0 +1,33 @@
+Supported platforms
+===================
+
+The |var-project-short| supports three primary platforms:
+
+- Linux
+- macOS
+- Windows
+
+Code examples are provided for each of these platforms throughout the |var-project-short| documentation. For example:
+
+.. tabs::
+
+    .. group-tab:: Linux
+
+        Linux-specific information
+
+    .. group-tab:: macOS
+
+        macOS-specific information
+
+    .. group-tab:: Windows
+
+        Windows-specific information
+
+Other platforms where |var-project-short| may work are:
```

Comment:
It may be more diplomatic to replace "may work" with "is supported with some caveats".

Reply:
Perhaps we can add a brief description of these caveats?

Reply:
This is something you need to talk to @ThijsSassen  or maybe @eboasson  about, they should know more about that.